setup.py

```diff
@@ -63,17 +63,17 @@
 
 _onnxruntime_deps = ["onnxruntime>=1.0.0"]
 _clip_deps = ["open_clip_torch==2.20.0"]
-supported_torch_version = "torch>=1.7.0,<=2.0"
+supported_torch_version = "torch>=1.7.0,<=2.1"
```

Comment:
could we do < rather than <= so we could pick up hotfixes? it should work even if 2.2 doesn't exist